pygmt/src/x2sys_init.py

```diff
@@ -87,7 +87,7 @@ def x2sys_init(tag, **kwargs):
         - **c** - Cartesian userdist or userdist/usertime
         - **e** - meters or m/s
         - **f** - feet or feet/s
-        - **k** - km or km/hr
+        - **k** - kilometers or km/hr
         - **m** - miles or miles/hr
```

Comment:
When stating a rate, should abbreviations be used or not? This uses both long unit names (miles/hr) and short unit names (m/s), so it should at least be standardized.

Reply:
Yep, I saw this and changed only the first column. The second column is a bit tricky when we want to use the short cuts:
```
        - **c** - Cartesian userdist or userdist/usertime
        - **e** - meters or m/s
        - **f** - feet or feet/s                     -> feet or ft/s
        - **k** - kilometers or km/hr
        - **m** - miles or miles/hr             -> miles or mi/hr
        - **n** - nautical miles or knots        -> velocity unit is missing or not available?, short cut for nautical miles?
        - **u** - survey feet or survey feet/s    -> short cut for survey feet?
```

Reply:
Hm. How to proceed here @GenericMappingTools/pygmt-maintainers:
(i) Leave as is, with non SI units in long forms
(ii) Change all units to long forms
(iii) Change all units to short cuts, please help with the short cuts for the non SI units

Reply:
I think that using abbreviations is fine when stating rates, since the long name is already stated.

Reply:
Changed the units of the velocities to their abbreviations in commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/b8f2103829bbf5e360027d9e051a601d1630f051.